src/ClockPanel.tsx

```diff
@@ -2,20 +2,22 @@ import { css } from '@emotion/css';
 import { PanelProps } from '@grafana/data';
 import { useTheme2 } from '@grafana/ui';
 import React, { useEffect, useMemo, useState } from 'react';
-import { ClockOptions, ClockRefresh } from './types';
+import { ClockOptions, ClockRefresh, DescriptionSource } from './types';
 
 // eslint-disable-next-line
 import { RenderDate } from 'components/RenderDate';
 import { RenderTime } from 'components/RenderTime';
 import { RenderZone } from 'components/RenderZone';
-import { Moment } from 'moment-timezone';
+import moment, { Moment } from 'moment-timezone';
```

Comment:
What is the purpose of changing this import?

On the line 59 you can just use `Moment`

Reply:
Yes, you are right!